CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Added
 - `"SCMP_ARCH_MIPS64N32"` to `ScmpArch::from_str()`.
+- `ScmpFilterContext::{get,set}_bad_arch_action()` to get/set the default action taken on a syscall for
```

Comment:
```suggestion
- `ScmpFilterContext::{get,set}_badarch_action()` to get/set the default action taken on a syscall for
```

Reply:
Sorry, I forgot to fix them, I'll fix it and squash the commits.